### PR TITLE
MF-1535 - Add API keys functions to SDK

### LINKED ISSUE #1535: Add API keys to SDK
**FEATURE REQUEST**

1. Is there an open issue addressing this request? If it does, please add a "+1" reaction to the
   existing issue, otherwise proceed to step 2.
No
2. Describe the feature you are requesting, as well as the possible use case(s) for it.
Add functions for API keys to SDK
3. Indicate the importance of this feature to you (must-have, should-have, nice-to-have).
should-have

### PR DESCRIPTION
**Pull request by fbugarski**
### What does this do?
Add API keys functions to SDK
### Which issue(s) does this PR fix/relate to?
Resolves #1535 

### Have you included tests for your changes?
No
### Did you document any new/modified functionality?
No
### Notes

### CODE REVIEW

File: pkg/sdk/go/sdk.go [hunk 1]
```diff
@@ -111,6 +110,21 @@ type Channel struct {
 	Metadata map[string]interface{} `json:"metadata,omitempty"`
 }
 
+//Member represents mainflux member.
```
Comment (drasko) [inline pkg/sdk/go/sdk.go]: Space after `//`

**Review by drasko — CHANGES_REQUESTED** [review]

**Comment by drasko** [comment]
@fbugarski there are conflicts on this branch that must be resolved

File: pkg/sdk/go/sdk.go [hunk 2]
```diff
@@ -109,12 +110,21 @@ type Channel struct {
 	Metadata map[string]interface{} `json:"metadata,omitempty"`
 }
 
-//Member represents mainflux member.
+// Member represents mainflux member.
```
Comment (drasko) [inline pkg/sdk/go/sdk.go]: This is unclear? What is `mainflux member`? You mean `group member`?

File: pkg/sdk/go/requests.go [hunk 3]
```diff
@@ -19,3 +25,32 @@ type ConnectionIDs struct {
...
 	ThingIDs   []string `json:"thing_ids"`
 }
+
+type issueKeyReq struct {
+	token    string
+	Type     uint32        `json:"type,omitempty"`
+	Duration time.Duration `json:"duration,omitempty"`
+}
+
+// It is not possible to issue Reset key using HTTP API.
```
Comment (drasko) [inline pkg/sdk/go/requests.go]: This is a confusing comment. This comment is supposed to describe what is implemented below, but looks like it does not.

File: pkg/sdk/go/sdk.go [hunk 4]
```diff
@@ -256,6 +266,15 @@ type SDK interface {
 
 	// RevokeCert revokes certificate with certID for thing with thingID
 	RevokeCert(thingID, certID, token string) error
+
+	// Issue issues a new Key, returning its token value alongside.
```
Comment (drasko) [inline pkg/sdk/go/sdk.go]: Do we need to capitalize `Key` in this comment? Maybe simple `key` would be more appropriate?

File: pkg/sdk/go/sdk.go [hunk 5]
```diff
@@ -256,6 +266,15 @@ type SDK interface {
...
+	Issue(token string, key Key) (issueKeyRes, error)
+
+	// Revoke removes the Key with the provided id that is issued by the user identified by the provided key.
+	Revoke(token, id string) error
+
+	// RetrieveKey retrieves data for the Key identified by the provided ID, that is issued by the user identified by the provided key.
```
Comment (drasko) [inline pkg/sdk/go/sdk.go]: In the upper comment you worte `id`, and here `ID`. Please stay consistent.

**Review by drasko — CHANGES_REQUESTED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Review by manuio — APPROVED** [review]
LGTM

**Review by dborovcanin — APPROVED** [review]
@fbugarski Please don't forget to open an issue for SDK API keys tests.

**Pull request merged**